base-notebook/Dockerfile

```diff
@@ -42,6 +42,7 @@ RUN apt-get update --yes && \
     wget \
     ca-certificates \
     sudo \
+    rsync \
```

Comment:
Let's not add more dependencies here to make images as small as possible.
I think `cp -R` or bare cp could be enough.

Reply:
I can definitely make that change. Thanks for the feedback!